<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditionMetadonnees</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources/Ressources.qrc">
    <normaloff>:/icons/icon-edition-metadonnees</normaloff>:/icons/icon-edition-metadonnees</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayoutMainWindow">
    <item row="0" column="0" colspan="7">
     <widget class="QTableWidget" name="tableWidget">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>600</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>Fichier</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Titre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Artiste</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Artiste de l'album</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Album</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Disque</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Piste</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Genre</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="labelBitRate">
      <property name="text">
       <string>Bit rate :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="comboBoxBitRate">
      <property name="currentIndex">
       <number>5</number>
      </property>
      <item>
       <property name="text">
        <string>32 kbit/s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>64 kbit/s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>96 kbit/s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>128 kbit/s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>160 kbit/s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>192 kbit/s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>224 kbit/s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>256 kbit/s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>288 kbit/s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>320 kbit/s</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QCheckBox" name="checkBoxEncodeFiles">
      <property name="text">
       <string>Re-encode files</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QCheckBox" name="checkBoxClearMetadata">
      <property name="text">
       <string>Clear metadata</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QCheckBox" name="checkBoxWriteMetadata">
      <property name="text">
       <string>Write metadata</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QCheckBox" name="checkBoxOrderFiles">
      <property name="text">
       <string>Re-order files</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1083</width>
     <height>38</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionProcess"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuInformation">
    <property name="title">
     <string>Informations</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuInformation"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../ressources/Ressources.qrc">
     <normaloff>:/icons/icon-action-new</normaloff>:/icons/icon-action-new</iconset>
   </property>
   <property name="text">
    <string>Créer une nouvelle édition de métadonnées...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="../ressources/Ressources.qrc">
     <normaloff>:/icons/icon-action-load</normaloff>:/icons/icon-action-load</iconset>
   </property>
   <property name="text">
    <string>Charger une édition de métadonnées existante...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../ressources/Ressources.qrc">
     <normaloff>:/icons/icon-action-save</normaloff>:/icons/icon-action-save</iconset>
   </property>
   <property name="text">
    <string>Sauvegarder l'édition de métadonnées courante...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionProcess">
   <property name="icon">
    <iconset resource="../ressources/Ressources.qrc">
     <normaloff>:/icons/icon-action-process</normaloff>:/icons/icon-action-process</iconset>
   </property>
   <property name="text">
    <string>Appliquer l'édition de métadonnées courante...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../ressources/Ressources.qrc">
     <normaloff>:/icons/icon-action-quit</normaloff>:/icons/icon-action-quit</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="icon">
    <iconset resource="../ressources/Ressources.qrc">
     <normaloff>:/icons/icon-action-documentation</normaloff>:/icons/icon-action-documentation</iconset>
   </property>
   <property name="text">
    <string>Documentation...</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../ressources/Ressources.qrc">
     <normaloff>:/icons/icon-action-about</normaloff>:/icons/icon-action-about</iconset>
   </property>
   <property name="text">
    <string>À propos de EditionMetadonnees...</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../ressources/Ressources.qrc"/>
 </resources>
 <connections/>
</ui>
